backend/metalit_challenge/api/auth.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+
+import jwt
+from django.conf import settings
+from django.contrib import auth
+from rest_framework import exceptions
+from rest_framework.authentication import BaseAuthentication
+
+from .serializers import UserSerializer
+
+from .models import User
+
+key = settings.JWT_SECRET_KEY
```

Comment:
@fadelananda could you rename it as jwt_key? so the variable name is more clear. Also don't forget to update all the referenced vars below.